install.sh

```diff
@@ -139,6 +182,7 @@ venv_setup ()
     source "${jm_source}/jmvenv/bin/activate" || return 1
     pip install --upgrade pip
     pip install --upgrade setuptools
+    pip install mako
```

Comment:
the C-lightning v0.10.2 build also needs 
`pip install mrkd`
otherwise getting the error:
```
cc tools/test/enum.c
mrkd doc/lightning-cli.1.md
/bin/sh: 1: mrkd: not found
make: *** [doc/Makefile:107: doc/lightning-cli.1] Error 127
make: *** Waiting for unfinished jobs....
mrkd doc/lightningd.8.md
/bin/sh: 1: mrkd: not found
make: *** [doc/Makefile:107: doc/lightningd.8] Error 127
mrkd doc/lightningd-config.5.md
/bin/sh: 1: mrkd: not found
make: *** [doc/Makefile:107: doc/lightningd-config.5] Error 127
rm tools/test/test_gen.c.tmp.c
c-lightning was required, but not built. Exiting.
```

Reply:
I see, thanks .. I already had `mrkd` installed. Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000/commits/c2138f924b3b1fe2d342df94a827c576b3d7411b

Reply:
as mentioned in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000#issuecomment-1011326052
needed to add:
`pip install mistune==0.8.4`
also

Reply:
Also have this issue, adding `pip install mistune==0.8.4` solves this.